.github/workflows/ci.yml

```
@@ -2,208 +2,208 @@ name: CI/CD Pipeline
+          curl -f http://localhost:8000/pages/about.html || exit 1
+          curl -f http://localhost:8000/pages/contact.html || exit 1
+          curl -f http://localhost:8000/pages/tickets.html || exit 1
+          curl -f http://localhost:8000/pages/donations.html || exit 1
+
+          # Test 2025 festival pages
+          curl -f http://localhost:8000/pages/boulder-fest-2025-index.html || exit 1
+          curl -f http://localhost:8000/pages/boulder-fest-2025-artists.html || exit 1
+          curl -f http://localhost:8000/pages/boulder-fest-2025-schedule.html || exit 1
+          curl -f http://localhost:8000/pages/boulder-fest-2025-gallery.html || exit 1
+
+          # Test 2026 festival pages  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)